examples/lib/README.md

```diff
@@ -1,16 +1,15 @@
 # How to work with the library
 
 The complete library is arranged around area specific [`Context` objects](contexts.md),
-which bundle all the settings and supported [extension points](../../docs/ocm/interoperability.md#support-library)
-of the *Open Component Model*.
```

Comment:
Are we sure that we want to delete that link? We could also restore the file, if it is important. (Maybe it really just slipped through)

Reply:
I don't know...

Reply:
I think @morri-son was part of the migration to `ocm-spec`. Let's wait for some input from him.

Reply:
clarified with @morri-son - https://github.com/open-component-model/ocm/pull/871/commits/8b9a62e91b3f8d28a2f91c013028265339e8f8f7